docker-compose.yaml

```diff
@@ -28,10 +28,10 @@ volumes:
     #
     # If you're on Mac, make sure you comment the following lines
     # and leave it as a regular docker volume
-    # driver: local-persist
-    # driver_opts:
-    #  name: "paritydb"
-    #  mountpoint: "${PARITY_VOLUME_MOUNTPOINT}"
+    driver: local-persist
```

Comment:
@gsiuniak make sure this changes are not present in the PR (we want develop to have the driver commented to avoid issues with random people downloading the project)

Reply:
Fixed. Thank you. Strangely enough, I thought commented that out, but somehow un-commented it. Whoops.